src/main/java/org/broadinstitute/hellbender/engine/AbstractConcordanceWalker.java

```diff
@@ -70,9 +70,16 @@
     // Overriding the superclass method to favor the truth variants' dictionary
     @Override
     public final SAMSequenceDictionary getBestAvailableSequenceDictionary() {
+        initializeTruthVariantsIfNecessary();
```

Comment:
It's subtle, which is why I only put it in after discovering the null pointer exception.  `GATKTool` calls `getBestAvailableSequenceDictionary` inside `initializeIntervals` from `onStartup`.  Since this always occurs before we initialize truth variants anywhere else, I could just do so inside `getBestAvailableSequenceDictionary` but that's 1) obscure and 2) could break easily if the parent class changed.